tests/algorithms/test_bc.py

```diff
@@ -127,10 +127,11 @@ def test_smoke_bc_creation(
     rng: np.random.Generator,
     pytestconfig: pytest.Config,
 ):
+    assert pytestconfig.cache is not None
```

Comment:
We use `pytestconfig.cache` a bunch of times here so I like the idea of adding a temporary variable `cache = pytestconfig.cache` from https://github.com/HumanCompatibleAI/imitation/pull/594/files#diff-648af1fe52e7f75f6ddc457b062fe9ee7b456d9358c493cef1a3547f5e87785b -- consider cherry-picking that change?